### PR TITLE
[IMPROVEMENT] Show action sheet when tapping usernames, mentions, etc

### PR DESCRIPTION
**Pull request by cardoso**
@RocketChat/ios

- Instead of just opening the direct message without prompting the user, now we show an action sheet.
- Also fixes real name/username not displaying according server settings in member cells. (used in members list and reactions)

Preview images:

iPhone:

### CODE REVIEW

File: Rocket.Chat/Views/Cells/Chat/Info/MemberCell.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ struct MemberCellData {
 
     var nameText: String {
         let utcText = member.utcOffset != nil ? "(UTC \(member.utcOffset ?? 0))" : ""
-        return "\(member.name ?? "") \(utcText)"
+        return "\(member.displayName()) \(utcText)"
```
Comment (rafaelks) [inline Rocket.Chat/Views/Cells/Chat/Info/MemberCell.swift]: 👍

File: Rocket.Chat/Views/ActionSheets/UserActionSheetPresenter.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                controller.dismiss(animated: true, completion: nil)
+                self?.dismiss(animated: true, completion: nil)
+            }
+        }))
+
+        controller.addAction(UIAlertAction(title: localized("global.cancel"), style: .cancel))
+
+        present(controller, animated: true, completion: nil)
+    }
+}
```
Comment (rafaelks) [inline Rocket.Chat/Views/ActionSheets/UserActionSheetPresenter.swift]: 👍

**Comment by rafaelks** [comment]
@cardoso I really liked the approach and I think this is going to be useful to our next release, where we're going to have more options on the user, like open the profile (in the future: mute, make admin, block, etc). 👍

**Comment by cardoso** [comment]
Exactly @rafaelks 

I made this after I looked our issue with most the +1s : https://github.com/RocketChat/Rocket.Chat.iOS/issues/1319

That's probably coming next since we already have the API for that https://rocket.chat/docs/developer-guides/rest-api/channels/kick/#channel-kick

**Comment by filipealva** [comment]
@cardoso This is a great approach! In the future, we can use a profile-like modally presented view controller instead of the action sheet, that way we will be able to provide more information about the user along with all the possible actions in a richer way.

**Pull request merged**